uprotocol/uattributes.proto

```diff
@@ -60,19 +60,20 @@ message UAttributes {
     // https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/basics/qos.adoc[uProtocol Prioritization]
     UPriority priority = 5;
 
-    // TTL is How long this message should live for after it was generated (in milliseconds).
+    // TTL is How long this message should live for after it was generated (in milliseconds). If not present,
+    // the message is assumed to not timeout (should live forever)
     // Event expires when:
     // \$t_current > t_{id} + ttl\$
-    optional int32 ttl = 6;
+    optional uint32 ttl = 6;
     
     // Permission level per https://github.com/eclipse-uprotocol/uprotocol-spec/blob/main/basics/permissions.adoc[Permissions]
-    optional int32 permission_level = 7;
+    optional uint32 permission_level = 7;
 
     // Communication error attribute populated by uP-L2 dispatchers only when an error 
     // has occurred in the delivery of RPC request or response events. 
     // The contents of this attribute, if present, is the unsigned integer representation of
-    // UCode
-    optional int32 commstatus = 8;
+    // UCode. If the attribute is not present, there is no communication error
```

Comment:
I find it a little irritating that we refer to UCode values here but the type of the property is uint32 instead of UCode. Is there a particular reason for this? If so, I guess we should provide it here ...

Reply:
This is legacy that I dragged from CloudEvent, I fixed this now (it will be breaking change for the client libraries).